lightningd/netaddress.c

```diff
@@ -265,7 +265,15 @@ static bool guess_one_address(struct lightningd *ld,
         memcpy(addr->addr, &sin6.sin6_addr, addr->addrlen);
         break;
     }
-    case ADDR_TYPE_PADDING:
+	case ADDR_TYPE_TOR_V2:
+        log_debug(ld->log, "Tor v2 address, ignoring");
+        return false;
+
+	case ADDR_TYPE_TOR_V3:
+        log_debug(ld->log, "Tor V3 address, ignoring");
+        return false;
+
+	case ADDR_TYPE_PADDING:
```

Comment:
This seems to be a fight between expanded tabs and real tabs. In [STYLE.md](https://github.com/ElementsProject/lightning/blob/master/doc/STYLE.md) the [Linux coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html) is referred and in Linux sources one can see real TABs (the same you use in your patch). Just the inconsistency (the fact the file was originally written with spaces and is not changed completely in this patch) can be confusing sooner or later.